index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600&family=Roboto:ital,wght@0,100;0,400;0,500;0,700;0,900;1,100;1,400;1,500&display=swap');
+    </style>
+  </head>
+  <body>
+
+
+
+<section class="full-page">
+      
+<!------- Navbar & background video --------->
+  
+  <header>
+        <video class="back-video" muted loop autoplay controls playsinline>
+            <source src="videos/Video3.mp4" type="video/mp4">
+        </video>
+
+          <nav>
+              <img src="Images/Earth.png" class="logo">
+              
+              <ul class="nav-links">
+                <a href="#accordion" class="button-faq">FAQs</a>
+              </ul>
+          </nav>
+      </header>
+
+
+<!------- Content over video --------->
+  <div class="content">
```

Comment:
Maybe the div-tag that wrap around the whole site's content can be a main-tag? Just like how you used semantic elements for each section, you could utilise the main tag as well?